examples/catalog/lib/animated_list.dart

```diff
@@ -157,11 +165,11 @@ class CardItem extends StatelessWidget {
     @required this.animation,
     this.onTap,
     @required this.item,
-    this.selected = false
-  }) : assert(animation != null),
-       assert(item != null && item >= 0),
-       assert(selected != null),
-       super(key: key);
+    this.selected = false,
+  })  : assert(animation != null),
+        assert(item != null && item >= 0),
+        assert(selected != null),
+        super(key: key);
```

Comment:
the extra space here just makes this code look weird, i don't really see why it would be desireable.

Reply:
@munificent what is the reason behind the extra space here?

Reply:
note that the cart is actually already out of the barn on this in the package:flutter repo. A number of files that were probably auto-formatted in package:flutter use the two spaces convention here as well.

Reply:
the reason behind the extra space is explained in https://github.com/dart-lang/dart_style/pull/672#issuecomment-367427958

Reply:
summary from the bug is the concern was about having an odd numbered line indents instead of keeping things a factor of 2. I'm having trouble seeing why the extra space is worth it to keep the indent even.

Reply:
Take a look at the output again. I've modified the formatter to not generate the double spaces. Thanks @a14n for the link to your original CL that only added 1 space.

Reply:
More explanation here: https://github.com/dart-lang/dart_style/issues/658